### ISSUE TITLE
Cannot set config for admin/url/use_custom using Magento CLI

### ISSUE THREAD

**Issue by tylers-username** [magento/magento2#23232]
### Preconditions (*)

1. Magento 2.2.4
2. OS: Debian (Docker PHP)

### Steps to reproduce (*)

1.Set the custom admin URL: `bin/magento config:set admin/url/custom "https://admin.example.com/"`
2. Enable the new custom URL path: `bin/magento config:set admin/url/use_custom 1`

### Expected result (*)

1. Given that an admin URL has been defined, I expected the `admin/url/use_custom` to pass validation and to be added to `core_config_data` table with a value of 1.

### Actual result (*)

1. The Magento Cli throws the exception message:
    > Please specify the admin custom URL.

**Comment by tylerssn** [comment]
@magento give me 2.2.4

**Comment by tylers-username** [comment]
@magento give me 2.2.4 instance

**Comment by magento-engcom-team** [comment]
Hi @tylerssn. Thank you for your request. I'm working on Magento 2.2.4 instance for you

**Comment by magento-engcom-team** [comment]
Hi @tylerssn, here is your [Magento instance](https://i-23232-2-2-4.instances.magento-community.engineering).
Admin access: [https://i-23232-2-2-4.instances.magento-community.engineering/admin](https://i-23232-2-2-4.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by tylers-username** [comment]
Can only confirm locally on a fresh 2.2.4 instance since I do not have shell access to the staging environment.

**Comment by sunilit42** [comment]
@magento give me 2.3.1 instance

**Comment by magento-engcom-team** [comment]
Hi @sunilit42. Thank you for your request. I'm working on Magento 2.3.1 instance for you

**Comment by magento-engcom-team** [comment]
Hi @sunilit42, here is your [Magento instance](https://i-23232-2-3-1.instances.magento-community.engineering).
Admin access: [https://i-23232-2-3-1.instances.magento-community.engineering/admin](https://i-23232-2-3-1.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by tylerssn** [comment]
To add to this, when I run each of the following commands at scope, they replace the previous commands value instead of creating a new entry:

```bash
bin/magento config:set web/cookie/cookie_path "/us" --scope website --scope-code base \
 && bin/magento config:set web/cookie/cookie_path "/ca" --scope website --scope-code canada \
 && bin/magento config:set web/cookie/cookie_path "/fr-ca" --scope stores --scope-code fr-ca \
 && bin/magento config:set web/cookie/cookie_path "/spa" --scope website --scope-code latin_america
```

**Comment by magento-engcom-team** [comment]
:white_check_mark: Confirmed by @shikhamis11 
 Thank you for verifying the issue. Based on the provided information internal tickets `MC-17525`, `MC-17526` were created

**Issue Available**: @shikhamis11, _You will be automatically unassigned. Contributors/Maintainers can claim this issue to continue. To reclaim and continue work, reassign the ticket to yourself._

**Comment by shikhamis11** [comment]
hello @tylerssn ,Thank you for your report.
This issue is already fixed in PR #22281 , #22282 It will be available in next 2.2.x and 2.3.x releases.
